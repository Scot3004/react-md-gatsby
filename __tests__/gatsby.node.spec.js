import gatsbyNode from '../gatsby-node';

describe('Gastby config', () => {
  it('should be equals to snapshot', () => {
    expect(gatsbyNode).toMatchSnapshot();
  });
});
