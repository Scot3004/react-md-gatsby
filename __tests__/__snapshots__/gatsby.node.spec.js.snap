// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gastby config should be equals to snapshot 1`] = `
Object {
  "createPages": [Function],
  "modifyWebpackConfig": [Function],
  "onCreateNode": [Function],
  "setFieldsOnGraphQLNodeType": [Function],
}
`;
